source/api/image/3.0/index.md

```diff
@@ -667,31 +667,54 @@ A server that supports neither `sizeByW` or `sizeByWh` is only required to serve
 
 The set of features, formats and qualities supported is the union of those declared in the external profile document and those added by the `extra` properties. If a feature is not present in either the profile document or the `extraFeatures` property, then a client _MUST_ assume that the feature is not supported.
 
-### 5.7. Related Services
+### 5.7. Linking Properties
 
-The JSON response _MAY_ also reference external services that make additional functionality available to a viewer.
+The JSON response _MAY_ contain linking properties that reference external resources, including services that make additional functionality available to a viewer.  The linking properties have the same semantics and requirements as those in the [Presentation API][prezi-api].
 
 | Property   | Required? | Description |
 | ---------- | --------- | ----------- |
-| `service`  | Optional  | The `service` references external services that the client might interact with directly to gain additional information or functionality, for example a link to an authentication service. The value _MUST_ be a list of objects. See the [Service Registry][service-profiles] for known service types. |
+| `partOf`   | Optional  | A link to another resource that contains this image service, for example a link to a Canvas or Manifest. The value _MUST_ be a list of objects. Each item _MUST_ have the `id` and `type` properties, and _SHOULD_ have the `label` property. |
```

Comment:
contains or references?

Reply:
I think the Canvas or Manifest _contains_ -- that usefully distinguishes from the linking to the Canvas or Manifest